lib/repl.js

```diff
@@ -31,6 +31,8 @@ const rl = require('readline');
 const Console = require('console').Console;
 const domain = require('domain');
 const debug = util.debuglog('repl');
+const HISTORY_PATH = path.join(process.env.HOME || process.env.USERPROFILE,
+                               '.iojs_history');
```

Comment:
This change should not be made to the REPL module, it should be specific to iojs command line repl.